## Введение
- [Введение](readme.md)
- [#0 Стать аналитиком блокчейна](ch00/ch00-become-chain-analyst.md)
Ошибка: HTTPConnectionPool(host='192.168.88.82', port=11434): Max retries exceeded with url: /api/generate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f89b7bd57b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Ошибка: HTTPConnectionPool(host='192.168.88.82', port=11434): Max retries exceeded with url: /api/generate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdbb1b417b0>: Failed to establish a new connection: [Errno 111] Connection refused'))